Rebuild-Black-Business/RBB-Website
# Fix mixed content warning on about and business pages

# Describe your PR
These two images were loading via http rather than https, which is a no-no.  Switched them to use `//`, which will load with `http` or `https` based on how the page is loaded.  It will likely be `https` all the time anyway.

Related to #n/a
Fixes #browser console warning

## Pages/Interfaces that will change
`/businesses`
`/about`

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. check the preview on this branch, and load `/businesses` and `/allies`.  You should not see this warning:
![image](https://user-images.githubusercontent.com/1844496/84305615-5a490d80-ab28-11ea-8b55-d67937fc2c78.png)


### Additional notes
